associations/has_many_association.go

```diff
@@ -119,3 +120,51 @@ func (a *hasManyAssociation) AfterSetup() error {
 	}
 	return nil
 }
+
+func (a *hasManyAssociation) AfterProcess() AssociationStatement {
+	v := a.value
+	if v.Kind() == reflect.Ptr {
+		v = v.Elem()
+	}
+
+	otherIDField := "ID"
```

Comment:
"other" isn't an explicit name. Maybe "follower" as in the spec?

Reply:
using "belonging" to be more inline with "belongs_to"